peachjam/adapters/adapters.py

```diff
@@ -395,8 +395,9 @@ def fetch_relationships(self, imported_document, created_document):
                         object_work=amending_work,
                     )
 
-        if imported_document["commencements"]:
-            for commencement in imported_document["commencements"]:
+        commencements = created_document.commencements_json
```

Comment:
I'm not as familiar with this code so it might be obvious, but why are we now looking at the created document's commencements rather than the imported document's?

Reply:
the imported_document is the document metadata as received from indigo which originally contained commencements. Now the commencement data is a field on the document itself. We could  pull this code out into its own function like `get commencements`, do the api call there and update the created document and the relationships in one step, but I think its okay this way too.

Reply:
@actlikewill Thanks for the explanation